docs/reverse-proxy.md

```diff
@@ -34,8 +34,8 @@ Apache virtual server configuration
         ProxyRequests           Off
         ProxyPreserveHost       On
         AllowEncodedSlashes     NoDecode
-        ProxyPass               /       http://127.0.0.1:4873 nocanon
-        ProxyPassReverse        /       http://127.0.0.1:4873
+        ProxyPass               /       http://127.0.0.1:4873/ nocanon
```

Comment:
That's right https://github.com/verdaccio/docker-examples/blob/master/apache-verdaccio/apache_proxy/conf/000-default.conf#L14